services/brig/src/Brig/CanonicalInterpreter.hs

```diff
@@ -48,9 +55,18 @@ import Wire.Sem.Logger.TinyLog (loggerToTinyLog)
 import Wire.Sem.Now (Now)
 import Wire.Sem.Now.IO (nowToIOAction)
 import Wire.Sem.Paging.Cassandra (InternalPaging)
+import Wire.UserStore
+import Wire.UserStore.Cassandra
+import Wire.UserSubsystem
+import Wire.UserSubsystem.Interpreter
 
 type BrigCanonicalEffects =
-  '[ SFT,
+  '[ UserSubsystem,
+     DeleteQueue,
+     Error Wire.API.Federation.Error.FederationError,
+     Wire.FederationAPIAccess.FederationAPIAccess Wire.API.Federation.Client.FederatorClient,
```

Comment:
```suggestion
     FederationAPIAccess FederatorClient,
```
I suppose we can import these things such that we can write the much shorter version.